### PR TITLE
[W4.11e][T09-4]Yeo Zhuan Yu

### PR DESCRIPTION
**Pull request by ZhuanYu**

### CODE REVIEW

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -10,6 +10,30 @@
...
+        assertNotNull(1, 2);
+
+        // some null objects
+        assertAnyNull(null, "abc");
+        assertAnyNull("abc", "", "abc", null);
+        assertAnyNull("", "abc", null, "abc");
+        assertAnyNull(1, null);
+        assertAnyNull(null, 1, Integer.valueOf(1));
+        assertAnyNull(null, null);
+        assertAnyNull(null, "a", "b", null);
```
Comment (eugenepeh) [inline test/java/seedu/addressbook/common/UtilsTest.java:35]: Sometimes when adding test cases, think about how the method treat this new test case differently from the other existing ones.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -36,6 +60,14 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    private void assertAnyNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
```
Comment (eugenepeh) [inline test/java/seedu/addressbook/common/UtilsTest.java:64]: No need to abstract the `assertTrue(Utils.isAnyNull(...))` as separate method, because asserting the result of the method itself is fine.

Although existing test `assertAreUnique` does that, but that is to prevent `Arrays.asList(...)` from flying everywhere.

**Review by eugenepeh — COMMENTED** [review]
Good job on adding both positive and negative test cases, as well as empty and null scenario.

Some comments added. Please close the PR after reading the comments.
